src/MatrixClientPeg.js

```diff
@@ -223,7 +223,7 @@ class MatrixClientPeg {
         if (SettingsStore.isFeatureEnabled("feature_cross_signing")) {
             // TODO: Cross-signing keys are temporarily in memory only. A
             // separate task in the cross-signing project will build from here.
```

Comment:
We should probably make sure we save them to SSSS in the same PR to save people from getting half setup cross-signing if they turn the labs flag on & push the button.

Reply:
The JS SDK now saves them to SSSS when created. I have updated this comment to match, though it does make me wonder whether these callbacks are still needed.